### PR TITLE
Added logit and logistic functions to the calculator (Maths Option)

### LINKED ISSUE #5211: Add logit and logistic functions to the calculator
In the workshop we had data that looked like pervcentages (0 to 100) or proportions (0 to 1).  They are percentage cloud cover each day.
For analysis it would be good to be able to calculate the logit transform.  This is available (sort of) in the stats package, where it is called qlogis.  The inverse is called plogis.

I suggest these are 2 new keys at the bottom of the maths keyboard.  They can be called logit and logistic.
The logit should however implement the **logit funtion in the car package**.  This allows for proportions or percentages.  It also copes with values that are exactly 0 and 1.  Very nice.  The car package is (I think) already installed (at least it is on my machine.
The inverse (Logistic key) can call the plogis function from the stats package.  That isn't available in the car package. 

I have made it a 4.30 milestone, not because it is urgent, but just because it is simple.  It can change if there is too much other work.

### PR DESCRIPTION
**Pull request by Wycklife**
This fixes #5211 
@africanmathsinitiative/developers This is ready for review.

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons  This looks good and is ready for you.

File: instat/ucrCalculator.vb [hunk 1]
```diff
@@ -1122,6 +1121,20 @@ Public Class ucrCalculator
...
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("car::logit(p, percents = Range.p[2] > 1, adjust)", 24)
+        Else
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("car::logit()", 1)
+        End If
+    End Sub
+    Private Sub cmdLogistic_Click(sender As Object, e As EventArgs) Handles cmdLogistic.Click
+        If chkShowArguments.Checked Then
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("stats::plogis(q, Location = 0, Scale() = 1, lower.tail = True, Log.p = False))", 50)
+        Else
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("stats::logistic()", 1)
```
Comment (dannyparsons) [inline instat/ucrCalculator.vb]: Why are these two different functions here?

Comment (Wycklife) [reply instat/ucrCalculator.vb]: @dannyparsons I'm going through it to confirm

**Comment by Wycklife** [comment]
@dannyparsons This is now ready to be merged

**Pull request merged**